scheduler/pkg/store/experiment/db.go

```diff
@@ -73,13 +76,46 @@ func (edb *ExperimentDBManager) restore(startExperimentCb func(*Experiment) erro
 					return err
 				}
 				experiment := CreateExperimentFromRequest(&snapshot)
+				if experiment.Deleted {
```

Comment:
this is the crux of the change. we store now `deleted` and on restoring we just add the experiment to the in-memory store without (re)starting it.